tests/sentry/utils/test_auth.py

```diff
@@ -170,3 +176,17 @@ def test_sso_expiry_default():
 def test_sso_expiry_from_env():
     value = sentry.utils.auth._sso_expiry_from_env("20")
     assert value == timedelta(seconds=20)
+
+
+def test_construct_link_with_query():
+    path = "foobar"
+    query_params = {"biz": "baz"}
+    expected_path = "foobar?biz=baz"
+
+    assert construct_link_with_query(path=path, query_params=query_params) == expected_path
+
+    path = "foobar"
```

Comment:
This should be in a separate test IMO given it's testing different behavior

Reply:
I'll add notes to each of these.

since we're not in a class here, i'm just opting for a simple quick test 😅